gpu/amd_linux.go

```diff
@@ -32,19 +33,76 @@ const (
 	DRMDeviceDirGlob   = "/sys/class/drm/card*/device"
 	DRMTotalMemoryFile = "mem_info_vram_total"
 	DRMUsedMemoryFile  = "mem_info_vram_used"
+	DRMTotalMemoryFileGTT = "mem_info_gtt_total"
+	DRMUsedMemoryFileGTT  = "mem_info_gtt_used"
 
 	// In hex; properties file is in decimal
 	DRMUniqueIDFile = "unique_id"
 	DRMVendorFile   = "vendor"
 	DRMDeviceFile   = "device"
+
+	//AMD APU RDNA2 and RDNA3 for GTT memory
+	GFX1103 = "gfx1103" //890m, 780m, 760m, 740m GPU RDNA3 
+	GFX1037 = "gfx1037" //610M GPU RDNA2
+    	GFX1035 = "gfx1035" //680m, 660m GPU RDNA2
+	GFX1033 = "gfx1033" //Van Gogh RDNA2
+	GFX1036 = "gfx1036" //RDNA2
+	GFX1151 = "gfx1151" //RDNA3+
+	GFX1152 = "gfx1152" //RDNA3+
```

Comment:
may we add MI300A to?

Reply:
added, can you test it?

Reply:
Same here. I will remove 	
GFX941 = "gfx941" //MI300X, CDNA3
GFX942 = "gfx942" //MI300X, MI300A CDNA3
for now